tests/integration/apigateway/test_apigateway_basic.py

```diff
@@ -812,95 +809,6 @@ def test_api_gateway_authorizer_crud(self, aws_client):
                 restApiId=self.TEST_API_GATEWAY_ID, authorizerId=authorizer_id
             )
 
-    def test_apigateway_with_lambda_integration(self, create_rest_apigw, aws_client):
```

Comment:
this was a complete removal or a move?

Reply:
This was move and migration to be AWS validated to `test_apigateway_lambda.py` file, under the name `test_lambda_aws_integration_with_request_template`